rpc-rs/src/provider.rs

```diff
@@ -513,16 +510,19 @@ impl HostBridge {
                 let shutmsg: ShutdownMessage = serde_json::from_slice(&payload).unwrap_or_default();
                 // Backwards compatibility - if no host (or payload) is supplied, default
                 // to shutting down unconditionally
+                if shutmsg.host_id.is_empty() {
+                    warn!("Please upgrade your wasmcloud host to >= 0.58.3 if you use lattices with multiple hosts.")
```

Comment:
Pretty sure this needs to be 0.59.0

Reply:
I got this number from https://github.com/wasmCloud/wasmcloud-otp/pull/494/files where the host id was added to the shutdown message and version changed to 0.58.3. So any release version >= 0.58.3 should be good.

If we're about to release 0.59.0 there's no reason not to suggest the update.